package.json

```diff
@@ -19,7 +19,8 @@
     "test:title": "truffle test --network ganache ./test/token/CodexTitle.test.js",
     "test:proxy": "truffle test --network ganache ./test/token/CodexTitleProxy.test.js",
     "test:proxy-extended": "truffle test --network ganache ./test/token/CodexTitleProxy.extended.test.js",
-    "test": "truffle test --network ganache",
```

Comment:
My only suggestion here would be to give TravisCI it's own test command, just to make it possible to run all tests locally in case you wanted to run them before pushing changes. Something like:

```
"test": "truffle test --network ganache",
"ganache-cli": "ganache-cli > /dev/null &",
"test:travis-ci": "npm run ganache-cli && truffle test"
```

Reply:
Yeah, I was going back and forth on this but probably makes sense to leave a cmd in there for local testing too.